### PR TITLE
Update Cover and Intro Page

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Update Cover and Intro Page

### CODE REVIEW

**Comment by ardalis** [comment]
@nishanil Fixed PR so it doesn't include previous commits.

File: docs/architecture/cloud-native/index.md [hunk 1]
```diff
@@ -41,33 +41,35 @@ Author:
...
-The audience for this guide is mainly developers, development leads, and architects who are interested in building modern web applications using Microsoft technologies and services in the cloud.
+The audience for this guide is mainly developers, development leads, and architects who are interested in learning how to build applications designed for the cloud.
 
-A secondary audience is technical decision makers who are already familiar ASP.NET or Azure and are looking for information on whether it makes sense to upgrade to ASP.NET Core for new or existing projects.
+A secondary audience is technical decision-makers who plan to choose whether to build their applications using a cloud-native approach.
 
 ## How you can use this guide
 
-**UPDATE THIS**
+This guide begins by defining cloud-native and introducing a reference application built using cloud-native principles and technologies. Beyond these first two chapters, the rest of the book is broken up into specific chapters focused on topics common to most cloud-native applications. You can jump to any of these chapters to learn about cloud-native approaches to:
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]:
```suggestion
This guide begins by defining cloud native and introducing a reference application built using cloud-native principles and technologies. Beyond these first two chapters, the rest of the book is broken up into specific chapters focused on topics common to most cloud-native applications. You can jump to any of these chapters to learn about cloud-native approaches to:
```

File: docs/architecture/cloud-native/index.md [hunk 2]
```diff
@@ -41,33 +41,35 @@ Author:
...
+- Data and data access
+- Communication patterns
+- Scaling and scalability
+- Application resiliency
+- Monitoring and health
+- Identity and security
+- DevOps
 
-Feel free to forward this guide to your team to help ensure a common understanding of these considerations and opportunities. Having everybody working from a common set of terminology and underlying principles helps ensure consistent application of architectural patterns and practices.
+This guide is available both in PDF form and online. Feel free to forward this document or links to its online version to your team to help ensure common understanding of these topics. Most of these topics benefit from a consistent understanding of the underlying principles and patterns, as well as the tradeoffs involved in decisions related to these topics. Our goal with this document is to equip teams and their leaders with the information they need to make well-informed decisions for their applications' architecture, development, and hosting.
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]: missing hyphen in trade-offs

```suggestion
This guide is available both in PDF form and online. Feel free to forward this document or links to its online version to your team to help ensure common understanding of these topics. Most of these topics benefit from a consistent understanding of the underlying principles and patterns, as well as the trade-offs involved in decisions related to these topics. Our goal with this document is to equip teams and their leaders with the information they need to make well-informed decisions for their applications' architecture, development, and hosting.
```

File: docs/architecture/cloud-native/index.md [hunk 3]
```diff
@@ -41,33 +41,35 @@ Author:
...
 
-Editors:
-
-> **John Doe**
-
-## Introduction
+Participants and Reviewers:
 
-TODO
+> **Cesar de la Torre**, Sr. PM, .NET product team, Microsoft Corp.
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]: Can we be consistent if we're going to refer to Microsoft with or without Corp?

File: docs/architecture/cloud-native/index.md [hunk 4]
```diff
@@ -41,33 +41,35 @@ Author:
...
+>
+> **Nish Anil**, Sr. Program Manager, .NET team, Microsoft
 
-## Purpose
+Editors:
 
-TODO
+> **Maira Wenzel**, Content Developer, .NET team, Microsoft
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]: if you're adding seniority, you might as well add mine 😉 thanks for including me!

```suggestion
> **Maira Wenzel**, Sr. Content Developer, .NET team, Microsoft
```

File: docs/architecture/cloud-native/index.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ ms.date: 03/07/2019
 
 # Architecting Cloud Native .NET Applications for Azure
 
-![cover image](./media/cover.png)
+![cover image](./media/cover-preview.png)
```
Comment (mairaw) [inline docs/architecture/cloud-native/index.md]: instead of creating a new cover, shouldn't you just replace the temp file you have in the directory now?

**Review by mairaw — COMMENTED** [review]
Looks good. Left a few comments for you to consider.

File: docs/architecture/cloud-native/index.md (hunk 3 above)
Comment (nishanil) [reply docs/architecture/cloud-native/index.md]:
```suggestion
> **Cesar de la Torre**, Principal Program Manager, .NET team, Microsoft
```

File: docs/architecture/cloud-native/index.md [hunk 6]
```diff
@@ -41,33 +41,35 @@ Author:
...
+- Scaling and scalability
+- Application resiliency
+- Monitoring and health
+- Identity and security
+- DevOps
 
-Feel free to forward this guide to your team to help ensure a common understanding of these considerations and opportunities. Having everybody working from a common set of terminology and underlying principles helps ensure consistent application of architectural patterns and practices.
+This guide is available both in PDF form and online. Feel free to forward this document or links to its online version to your team to help ensure common understanding of these topics. Most of these topics benefit from a consistent understanding of the underlying principles and patterns, as well as the tradeoffs involved in decisions related to these topics. Our goal with this document is to equip teams and their leaders with the information they need to make well-informed decisions for their applications' architecture, development, and hosting.
 
 ## References
```
Comment (nishanil) [inline docs/architecture/cloud-native/index.md]: We should drop this.

File: docs/architecture/cloud-native/index.md (hunk 5 above)
Comment (nishanil) [reply docs/architecture/cloud-native/index.md]: I agree. It's easier to replace the image when the books go out of Preview.

**Comment by ardalis** [comment]
:shipit:

**Review by nishanil — APPROVED** [review]

**Pull request merged**